app/services/order_row_importer.rb

```diff
@@ -4,6 +4,27 @@
 
 class OrderRowImporter
 
+  module Overridable
+
+    def validate_custom_attributes
+      if field(:project_name).present? && project.nil?
+        add_error(:project_not_found)
+      end
+    end
+
+    private
+
+    def project
+      facility.projects.active.find_by name: field(:project_name)
```

Comment:
Don't know how this works, so maybe a dumb question: is this safe for schools that don't have `projects`?

Reply:
A very good point - thank you @LeticiaErrandonea !